apps/evalite-ui/app/routes/eval.$name.tsx

```diff
@@ -1,373 +1,20 @@
-                          )}
-                        </>
-                      )}
-
-                      {result.scores.length === 0 && (
-                        <td className="border-l">
-                          <Wrapper>
-                            <Score
-                              score={0}
-                              isRunning={isRunningEval}
-                              resultStatus={result.status}
-                              evalStatus={possiblyRunningEvaluation.status}
-                              state={"first"}
-                            />
-                          </Wrapper>
-                        </td>
-                      )}
-                      {result.scores.map((scorer, index) => {
-                        const scoreInPreviousEvaluation =
-                          prevEvaluation?.results
-                            .find((r) => r.input === result.input)
-                            ?.scores.find((s) => s.name === scorer.name);
-                        return (
-                          <td
-                            key={scorer.id}
-                            className={cn(
-                              index === 0 && "border-l",
-                              "align-top"
-                            )}
-                          >
-                            <Wrapper>
-                              <Score
-                                score={scorer.score}
-                                isRunning={isRunningEval}
-                                resultStatus={result.status}
-                                evalStatus={possiblyRunningEvaluation.status}
-                                state={getScoreState(
-                                  scorer.score,
-                                  scoreInPreviousEvaluation?.score
-                                )}
-                              />
-                            </Wrapper>
-                          </td>
-                        );
-                      })}
-                    </TableRow>
-                  );
-                })}
-              </TableBody>
-            </Table>
-          </>
-        )}
-      </InnerPageLayout>
-      <div
-        className={cn(
-          "fixed top-0 z-20 h-svh border-l p-2 bg-sidebar overflow-auto",
-          "transition-[right] ease-linear shadow-lg duration-300",
-          "hidden w-full sm:block sm:right-[-100%] sm:w-[500px] md:w-[600px] lg:w-[800px]",
-          isTraceRoute && "block sm:right-0",
-          !isTraceRoute && ""
-        )}
-      >
-        <Outlet />
-      </div>
-    </>
-  );
-}
+import { createFileRoute } from "@tanstack/react-router";
+import { z } from "zod";
+import { zodValidator } from "@tanstack/zod-adapter";
+
+const searchSchema = z.object({
+  timestamp: z.string().optional(),
+});
+
+export const Route = createFileRoute("/eval/$name")({
+  validateSearch: zodValidator(searchSchema),
+  loaderDeps: ({ search: { timestamp } }) => ({
```

Comment:
Needed to pass the timestamp onto `loader` https://tanstack.com/router/latest/docs/framework/react/guide/data-loading#using-loaderdeps-to-access-search-params